config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
       success: An invitation has been sent to the new user's email address
       errors:
         already_exists: This email address is already being used, enter a unique one
-        invalid_email: Enter an email address in the correct format
+        invalid_email: address needs to be in the correct format
```

Comment:
Surely should start with a capital letter and be a complete sentence – the original might be slightly better?

Reply:
And should we be testing this case?

Reply:
This was written this way as the error message when its displayed starts with 'Email' so the whole sentence reads "Email address needs to be in the correct format"

Reply:
So why is this one different to the others e.g. `already_exists` starts "This email..".

Also doing string munging like that is bound to break translations (if we ever get to doing those) – you need to have whole sentences in one block if they are to be translateable. String concatenation fails in the general case because of word order differences between natural languages.